src/olympia/reviewers/tests/test_utils.py

```diff
@@ -2318,19 +2329,18 @@ def test_reject_multiple_versions_delayed_uses_original_user(self):
         self.addon.reload()
         assert self.addon.status == amo.STATUS_NULL
 
-        # The versions are not pending rejection.
-        for version in self.addon.versions.all():
-            assert version.pending_rejection is None
-            assert version.pending_rejection_by is None
```

Comment:
These lines didn't work if the aim of the test was to emulate how it would have been called via the `auto_reject` command - the pending rejection flags are cleared in the `auto_reject` command itself rather than here, and when `human_review` is `False` (which it would be via `auto_reject`) then they wouldn't be cleared either.

Unless I'm misinterpreting what behaviour the test was originally validating.

Reply:
I think this is just to prove the rejection wasn't delayed - multiple tests have that, here it's just as a safeguard. The test is way too long though so it's hard to follow regardless...